website/screens/components/wizard/code/examples/basicUsage.ts

```diff
@@ -0,0 +1,32 @@
+import { DxcWizard, DxcInset } from "@dxc-technology/halstack-react";
+
+const code = `() => {
+    return (
+      <DxcInset space="medium">
```

Comment:
In the rest of the examples, we are using space="large".

Reply:
Done!